src/services/__tests__/QueueService.int.spec.ts

```diff
@@ -1,14 +1,18 @@
+import * as faker from "faker";
 import { joinQueue } from "../QueueService";
 import { leaveQueue } from "../QueueService";
-import { BallChaserQueueBuilder, LeaderboardBuilder } from "../../../.jest/Builder";
+import { ActiveMatchBuilder, BallChaserQueueBuilder, LeaderboardBuilder } from "../../../.jest/Builder";
```

Comment:
Do you need ActiveMatchBuilder imported?

Reply:
good catch. fixed.